python/semantic_kernel/orchestration/sk_function.py

```diff
@@ -301,12 +287,12 @@ def invoke(
         # Handle "asyncio.run() cannot be called from a running event loop"
         if loop and loop.is_running():
             if self.is_semantic:
-                thread = self.RunThread(self._invoke_semantic_async(context, settings))
+                thread = RunThread(self._invoke_semantic_async(context, settings))
                 thread.start()
                 thread.join()
                 return thread.result
```

Comment:
can you conflate these 4 lines into a single line?

Reply:
Can do!